baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/ValidateConstantMessage.java

```diff
@@ -45,6 +46,19 @@
     private static final Matcher<ExpressionTree> VALIDATE_METHODS =
             MethodMatchers.staticMethod()
                     .onClassAny("org.apache.commons.lang3.Validate", "org.apache.commons.lang.Validate");
+    private static final Matcher<ExpressionTree> VALIDATE_METHODS_MESSAGE_THIRD_PARAM =
+            Matchers.anyOf(
+                    MethodMatchers.staticMethod()
+                            // all org.apache.commons.lang.Validate methods have the message as the second arg
```

Comment:
this check runs on methods from both `org.apache.commons.lang.Validate` and`org.apache.commons.lang3.Validate`, but I'm only including certain `lang3` methods here, because all the `lang` methods fall under the "normal" case where the message string is the second arg